frontend/src/components/Home.vue

```diff
@@ -149,7 +174,76 @@ export default {
       } else {
         return `$${price.toFixed(2)}`
       }
+    },
+    formatPricePadding(price) {
+      function pad(n, width, z) {
```

Comment:
Why define this as a function if you're only ever going to call it in one place?